rdbms/src/main/java/io/spine/server/storage/jdbc/aggregate/EventCountTable.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.storage.jdbc.aggregate;
+
+import com.google.common.collect.ImmutableList;
+import io.spine.server.entity.Entity;
+import io.spine.server.storage.jdbc.DataSourceWrapper;
+import io.spine.server.storage.jdbc.Sql;
+import io.spine.server.storage.jdbc.TableColumn;
+import io.spine.server.storage.jdbc.query.EntityTable;
+import io.spine.server.storage.jdbc.query.SelectQuery;
+import io.spine.server.storage.jdbc.query.WriteQuery;
+
+import java.util.List;
+
+import static io.spine.server.storage.jdbc.Sql.Type.ID;
+import static io.spine.server.storage.jdbc.Sql.Type.INT;
+import static io.spine.server.storage.jdbc.aggregate.EventCountTable.Column.id;
+
+/**
+ * A table for storing the
+ * {@link io.spine.server.aggregate.AggregateStorage#readEventCountAfterLastSnapshot(Object)
+ * event count after the last snapshot}.
+ *
+ * <p>This table exists for the performance reason. The event count after the snapshot can
+ * be retrieved from {@link AggregateEventRecordTable} using a query,
+ * but this approach is less efficient.
```

Comment:
`... is less efficient for a majority of RDBMS due to a massive table scanning involved.`

Reply:
Done.